README.md

```diff
@@ -5,9 +5,70 @@
   <br />
 </p>
 
-# Amplitude-Go
+<div align="center">
+
+[![Build](https://github.com/amplitude/analytics-go/actions/workflows/build.yml/badge.svg)](https://github.com/amplitude/analytics-go/actions/workflows/build.yml)
+[![go-doc](https://pkg.go.dev/badge/github.com/amplitude/analytics-go?utm_source=godoc)](https://pkg.go.dev/github.com/amplitude/analytics-go)
+
+</div>
+
+# Announcement 📣
+
+Amplitude is introducing [Go SDK](https://pkg.go.dev/github.com/amplitude/analytics-go). Compared to plain [HTTP V2 API](https://www.docs.developers.amplitude.com/analytics/apis/http-v2-api/), it provides improved developer experience, helps users instrument data more seamlessly and provides more control over data being instrumented using custom plugins. 
+
+To learn more about the new SDK, here are some useful links:
+
+* Go Packages: https://pkg.go.dev/github.com/amplitude/analytics-go
+* GitHub: https://github.com/amplitude/analytics-go
+* Documentation: https://www.docs.developers.amplitude.com/data/sdks/go/
+
+
+# Official Amplitude Go SDK
+
+This is Amplitude's latest and official version of Go SDK.
+
+## Installation 
+
+Install analytics-go using go get:
+
+```
+go get https://github.com/amplitude/analytics-go
+```
+
+## Usage
+```go
+package main
+
+import (
+	"github.com/amplitude/analytics-go/amplitude"
+)
+
+func main() {
+
+	config := amplitude.NewConfig("your-api-key")
+
+	client := amplitude.NewClient(config)
+
+	// Track a basic event
+	// One of UserID and DeviceID is required
+	client.Track(amplitude.Event{
+		EventType: "Button Clicked",
+		EventOptions: amplitude.EventOptions{
+			UserID: "user-id",
+			//DeviceID: "device-id",
+		},
+		// Optional event properties
+		//EventProperties: map[string]interface{}{"source": "notification"},
```

Comment:
i see it's because it's optional but seeing commented out code is a little confusing. we can provide multiple examples instead

Reply:
I saw one example uses comments for optionals and thought it would be good. But I cannot find where it is. Anyway just checked out other go package examples, they are the simplest example. So, It is better to remove these comments to erase potential confusion. 

Removed